src/cmd_line/commands/file.ts

```diff
@@ -102,7 +103,12 @@ export class FileCommand extends node.CommandBase {
             filePath = pathWithExt;
           } else {
             // create file
-            fs.closeSync(fs.openSync(filePath, 'w'));
+            if (this.arguments.dontCreateFile === true) {
+              vscode.window.showErrorMessage('File path does not exits');
```

Comment:
there's a function in `util.ts` for this, can you use that? Also output the filepath.